one_fm/overrides/leave_policy_assignment.py

```diff
@@ -70,6 +72,17 @@ def create_leave_allocation(self, annual_allocation, leave_details, date_of_join
 		return allocation.name, new_leaves_allocated
 	
 
+class LeaveAllocationOverride(LeaveAllocation):
```

Comment:
Kindly put this in it's own file, this file belongs to leave policy assignment

Reply:
Done with the changes and updated the same as suggested.